helm/prometheus-rules/templates/alerting-rules/certificate.all.rules.yml

```diff
@@ -34,11 +34,22 @@ spec:
         severity: page
         team: phoenix
         topic: cert-manager
+    - alert: KyvernoCertificateSecretWillExpireInLessThanTwoDays
+      annotations:
+        description: '{{`Kyverno Certificate stored in Secret {{ $labels.namespace }}/{{ $labels.name }} on {{ $labels.cluster_id }} will expire in less than two days.`}}'
+        opsrecipe: kyverno-certificate-secret-will-expire-in-less-than-two-days/
+      expr: (cert_exporter_secret_not_after{name=~".*kyverno.*"} - time()) < 2 * 24 * 60 * 60
+      labels:
+        area: kaas
+        cancel_if_outside_working_hours: "true"
+        severity: page
+        team: shield
```

Comment:
Those are not paging anyone, right? Today, `shield` team only has slack notifications AFAIK.

Reply:
Should it be `{{ include "providerTeam" . }}`? cc @stone-z

Reply:
I think provider teams should not be paged for Kyverno. Either shield does or make it notify only

Reply:
Our Atlas amigos are right, let's make it only notify

Reply:
Sure!